runtime/browser/xwalk_notification_manager_linux.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef XWALK_RUNTIME_BROWSER_XWALK_NOTIFICATION_MANAGER_LINUX_H_
+#define XWALK_RUNTIME_BROWSER_XWALK_NOTIFICATION_MANAGER_LINUX_H_
+
+#include <libnotify/notification.h>
+#include <libnotify/notify.h>
+
+#include <map>
+
+#include "base/callback.h"
+#include "base/containers/scoped_ptr_hash_map.h"
+#include "base/strings/string16.h"
+
+class GURL;
+class SkBitmap;
+
+namespace content {
+class BrowserContext;
+class DesktopNotificationDelegate;
+class RenderFrameHost;
+struct PlatformNotificationData;
+}
+
+namespace xwalk {
+
+class XWalkNotificationManagerLinux {
```

Comment:
ok. just put it in subfolder 'linux'.  I'll summarize a general layer when there is requirement.